### ISSUE TITLE
Increasing search results

### ISSUE THREAD

**Issue by BrownBlueGreen** [tasos-py/Search-Engines-Scraper#11]
Sorry if basic question, how do i change search results? when using bing there's ~170 results, google 10 results, ask ~60 results, not sure how to modify.

Thanks

**Comment by BrownBlueGreen** [comment]
I think i actually got it, so nevermind.

**Comment by BrownBlueGreen** [comment]
Also thanks for the crawler it's pretty nice

**Comment by tasos-py** [comment]
Thank you! Now, if you want to limit the number of results there are two ways to do it - 

Linit the number of search results pages. This is not guaranteed to give you an accurate number, because different search engines produce different numbers of results per page (ranging from 10 to 25, from what I remember), eg:  
~~~
engine = Google()
results = engine.search("my query", 2)
print(len(results))
~~~
This should produce about 20 results (2*10).  

Slice the collected results. `SearchResults` (the object returned from `SearchEngine.search()`) is slicable and you can use the `[]` operator to select the range of items you wish, eg  
~~~
engine = Google()
results = engine.search("my query", 2)
first_20_results = results.results()[:20]
~~~

**Comment by Durga260996** [comment]
@BrownBlueGreen I got similar results. Please let me know how you resolved it. Thank you.

**Comment by tasos-py** [comment]
The Google problem was fixed in issue #12
